HabitRPG/Base.lproj/Main.storyboard

```diff
@@ -209,7 +209,7 @@
                                         <rect key="frame" x="0.0" y="28" width="600" height="67"/>
                                         <autoresizingMask key="autoresizingMask"/>
                                         <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="J05-li-dNN" id="jVr-kD-zKG">
-                                            <rect key="frame" x="0.0" y="0.0" width="600" height="66"/>
+                                            <rect key="frame" x="0.0" y="0.0" width="600" height="66.5"/>
```

Comment:
@a-ayyash @vIiRuS can you help me understand why the .5 change in the story board file? my latest version of xcode will automatically remove the .5 after opening the storyboard file.